### DISCUSSION TITLE
Unable to dismiss payment alert, even after downgrading to free

### DISCUSSION THREAD

**Discussion by jrbgit** [community/community#57996] [New to GitHub]
Please help me as this is driving me insane.
I no longer want to pay for GitHub Pro. The credit card GitHub has on file is no longer valid (I lost it many months after paying for a year subscription. There is no way to make this card valid - and again, I have downgraded my account to free at least 30 times now but every time I log in, I am unable to dismiss this alert. 

My account is paid for and displays this message when I try to downgrade:  "You are downgrading to GitHub Free. After October 04, 2023, your account features and limits will change." 

Yet every day, all day, while using GitHub... This is at the top of EVERY PAGE and can not be dismissed: "We are having a problem billing your account. Please enter a new payment method or check with your payment provider for details on why the transaction failed. You can downgrade to GitHub Free in your Billing settings. You can always contact support"

The card was lost in **JANUARY** and I have the first email from GitHub in January when this started as proof. Which means I'm going on 6 months of being harassed every day telling me I have to pay, unable to downgrade, and quite frankly really annoying to explain to another dev when they see the message on my screen. This is border lining on harassment at this point. Fix your shit GitHub, and until then, never expect to get my credit card again. 
<img width="733" alt="Screenshot 2023-06-13 223333" src="https://github.com/community/community/assets/2849289/88afbd14-04b3-438f-859d-7eabe61983ee">

**Comment by jrbgit** [comment]
🤡

**Comment by jrbgit** [comment]
"The plan change was successful. @jrbgit will be updated to the free yearly plan."

**Comment by jrbgit** [comment]
Can't remove a card on file either... add that to this.

**Comment by queenofcorgis** [comment]
Hi @jrbgit ! Ah that does sound frustrating, have you opened a ticket on our [Support page](https://support.github.com/contact)? They're the best folks to handle account-specific issues like this one.

**Comment by dinvlad** [comment]
I'm just experiencing this on an org I'm an admin on, but I'm not a billing decision-maker there. It's showing up on any repo in every other org however, and I'm unable to dismiss it. It's hard to understate how intrusive this is 😔
